CMakePresets.json

```diff
@@ -125,7 +127,9 @@
       "name": "clang-tidy-unix-base",
       "cacheVariables": {
         "CMAKE_C_CLANG_TIDY": "$env{CLANG_TIDY}",
-        "CMAKE_CXX_CLANG_TIDY": "$env{CLANG_TIDY}"
+        "CMAKE_CXX_CLANG_TIDY": "$env{CLANG_TIDY}",
```

Comment:
it's probably good to also do the `clang-tidy` the same way you do `clang`/`clang++` using symlink